api/oci/extensions/repositories/ocireg/repository.go

```diff
@@ -13,12 +13,12 @@ import (
 	"github.com/mandelsoft/logging"
 
 	"ocm.software/ocm/api/credentials"
-	"ocm.software/ocm/api/credentials/builtin/oci/identity"
 	"ocm.software/ocm/api/datacontext/attrs/rootcertsattr"
 	"ocm.software/ocm/api/oci/artdesc"
 	"ocm.software/ocm/api/oci/cpi"
 	"ocm.software/ocm/api/tech/docker"
 	"ocm.software/ocm/api/tech/docker/resolve"
+	identity2 "ocm.software/ocm/api/tech/oci/identity"
```

Comment:
no alias needed here

Reply:
jow, this goland, whenever you rename a package, you sometime get such renamed package aliases.

Reply:
changed